modules/src/applications/ics20_fungible_token_transfer/mod.rs

```diff
@@ -1,14 +1,21 @@
 //! ICS 20: Token Transfer implementation allows for multi-chain denomination handling, which
 //! constitutes a "fungible token transfer bridge module" between the IBC routing module and an
 //! asset tracking module.
+pub mod acknowledgement;
+pub mod address;
 pub mod context;
+pub mod denom;
 pub mod error;
+pub mod events;
 pub mod msgs;
+pub mod packet;
 pub mod relay_application_logic;
 
-mod denom;
 pub use denom::*;
 
+/// Module identifier for the ICS20 application.
+pub const MODULE_ID_STR: &str = "transfer";
+
 /// The port identifier that the ICS20 applications
 /// typically bind with.
 pub const PORT_ID: &str = "transfer";
```

Comment:
can we make this `PORT_ID_STR`?

Reply:
Done! eff458c2c03a7ac803f941968be521397f5bfe20